test/fixtures/discovery/cloud_discovery_aws.json

```diff
@@ -1,8 +1,17 @@
 {
-  "discovery_type": "cloud_discovery",
-  "agent_id": "0a055c90-4cb6-54ce-ac9c-ae3fedaf40d4",
-  "payload": {
-    "Provider": "aws",
-    "Metadata": null
-  }
+	"discovery_type": "cloud_discovery",
```

Comment:
different linting? :smile:

Reply:
vscode insists in adding 4 spaces tab...
I will edit the file with other editor by now...